src/controllers/tenants.cr

```diff
@@ -1,7 +1,7 @@
 class Tenants < Application
   base "/api/staff/v1/tenants"
 
-  before_action :admin_only
+  before_action :admin_only, except: [:current_limits, :show_limits]
```

Comment:
I would add a test for this to ensure any user can access

Reply:
probably no need to test for this as it's being tested at the spider-gazelle level